webapp/src/components/ParcelTags/ParcelTags.js

```diff
@@ -21,6 +22,10 @@ export default class ParcelTags extends React.PureComponent {
     showDetails: false
   }
 
+  isParcel() {
+    return this.props.parcel
```

Comment:
I'd return `!!this.props.parcel` here, to comply to the `isXXX returns boolean` convention